### PR TITLE
Adding example for conversion with user identifiers.

### PR DESCRIPTION
**Pull request by mcloonan**
Change-Id: I766f3bb64c3d6348e81dac5d407e56cc091e5fac

### CODE REVIEW

**Review by abraham — DISMISSED** [review]

File: examples/remarketing/upload_conversion_with_identifiers.rb [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    unless order_id.nil?
+      cc.order_id = order_id
+    end
+
+    # Creates a user identifier using the hashed email address, using the
+    # normalize and hash method specifically for email addresses.
+    # If using a phone number, use the normalize_and_hash method instead.
+    cc.user_identifiers << client.resource.user_identifier do |id|
+      id.hashed_email = normalize_and_hash_email(email_address)
+      id.user_identifier_source = :FIRST_PARTY
```
Comment (jradcliff) [inline examples/remarketing/upload_conversion_with_identifiers.rb:50]: nit: please add:

```
# Optional: Specifies the user identifier source.
```

**Review by jradcliff — DISMISSED** [review]

**Review by abraham — APPROVED** [review]

**Review by jradcliff — DISMISSED** [review]

File: examples/remarketing/upload_conversion_with_identifiers.rb [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    conversions: [click_conversion],
+    # Partial failure must be true.
+    partial_failure: true,
+  )
+
+  if response.partial_failure_error
+    puts "Partial failure encountered: #{response.partial_failure_error.message}"
+  else
+    result = response.results.first
+    puts "Uploaded click conversion that happened at #{result.conversion_date_time} " \
```
Comment (jradcliff) [inline examples/remarketing/upload_conversion_with_identifiers.rb:66]: @AnashOommen Just caught that this shouldn't be using GCLID. I'll give you an updated example.

Comment (jradcliff) [reply examples/remarketing/upload_conversion_with_identifiers.rb:66]: Please remove the `from Google Click ID #{result.gclid}` portion, then this should be good.

Thanks

**Review by AnashOommen — APPROVED** [review]
LGTM pending comments from Josh.

**Review by jradcliff — APPROVED** [review]

**Review by abraham — APPROVED** [review]

**Pull request merged**